ext/DynamicQuantitiesMakieExt.jl

```diff
@@ -0,0 +1,171 @@
+module DynamicQuantitiesMakieExt
+
+using DynamicQuantities: UnionAbstractQuantity, ustrip, dimension
+using TestItems: @testitem
+
+import Makie as M
+
+M.expand_dimensions(::M.PointBased, y::AbstractVector{<:UnionAbstractQuantity}) = (keys(y), y)
+M.create_dim_conversion(::Type{<:UnionAbstractQuantity}) = DQConversion()
+M.MakieCore.should_dim_convert(::Type{<:UnionAbstractQuantity}) = true
+
+unit_string(quantity::UnionAbstractQuantity) = string(dimension(quantity))
+
+function unit_convert(::M.Automatic, x)
+    x
+end
+
+function unit_convert(quantity::UnionAbstractQuantity, x::AbstractArray)
+    # Note: unit_convert.(Ref(quantity), x) currently causes broadcasting error for `QuantityArray`s
+    map(Base.Fix1(unit_convert, quantity), x)
+end
+
+function unit_convert(quantity::UnionAbstractQuantity, value)
+    conv = ustrip(quantity, value)
+    return Float64(conv)
+end
+
+# TODO: Maybe only allow symbolic units to avoid bugs?
```

Comment:
Status of this TODO?

Reply:
sg, I've also updated the docs accordingly ef2d0d8